MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -335,7 +335,7 @@ extension UIDevice {
             var systemInfo = utsname()
             uname(&systemInfo)
             let machineMirror = Mirror(reflecting: systemInfo.machine)
-            let identifier = machineMirror.children.reduce("") { identifier, element in
+            let identifier: String = machineMirror.children.reduce("") { (identifier: String, element: Mirror.Child) in
```

Comment:
Wouldn’t it be better to declare the return type of the closure instead of the type of the local variable?

Reply:
Yeah, we could return this variable right away instead.